### PR TITLE
PETOSS-561 Update existing tests in Dotnet SDK to work with PRISM Mock Server

### PR DESCRIPTION
**Pull request by vigneshk-tw**
…

### CODE REVIEW

File: Xero.NetStandard.OAuth2/Api/FinanceApi.cs [hunk 1]
```diff
@@ -1458,7 +1458,7 @@ public async System.Threading.Tasks.Task<IncomeByContactResponse> GetFinancialSt
 
             if (includeManualJournals != null)
             {
-              requestOptions.QueryParameters.Add("includeManualJournals", includeManualJournals.ToString());
+              requestOptions.QueryParameters.Add("includeManualJournals", includeManualJournals.ToString().ToLower());
```
Comment (manishT72) [inline Xero.NetStandard.OAuth2/Api/FinanceApi.cs:1461]: why are we setting it to lower() ?

File: Xero.NetStandard.OAuth2/Model/PayrollUk/Employment.cs [hunk 2]
```diff
@@ -123,6 +123,13 @@ public enum NiCategoryEnum
         /// <value>The NI Category of the employee</value>
         [DataMember(Name="niCategory", EmitDefaultValue=false)]
         public NiCategoryEnum NiCategory { get; set; }
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Employment" /> class.
+        /// </summary>
+        [JsonConstructorAttribute]
+        public Employment()
```
Comment (manishT72) [inline Xero.NetStandard.OAuth2/Model/PayrollUk/Employment.cs:130]: why are these classes generating now. We just fixed tests, let's discuss.

**Review by manishT72 — CHANGES_REQUESTED** [review]

File: Xero.NetStandard.OAuth2/Model/PayrollUk/Employment.cs (hunk 2 above)
Comment (vigneshk-tw) [reply Xero.NetStandard.OAuth2/Model/PayrollUk/Employment.cs:130]: These classes are generated from Open API, not from the code-gen. we've corrected some required fields and after that change these classes has generated

File: Xero.NetStandard.OAuth2/Api/FinanceApi.cs (hunk 1 above)
Comment (vigneshk-tw) [reply Xero.NetStandard.OAuth2/Api/FinanceApi.cs:1461]: when we convert a boolean value to string, it is taking the value as **T**rue or **F**alse.
PRISM is not able to understand the values if casing is wrong, for that reason we need to convert the values to lower

**Review by manishT72 — APPROVED** [review]

**Pull request merged**